### PR TITLE
Add note about 7.2.0 compat

### PR DESCRIPTION
**Pull request by assertchris**
Discussed https://github.com/laravel/framework/issues/22162#issuecomment-348185250

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -17,13 +17,15 @@
...
 <div class="content-list" markdown="1">
-- PHP >= 5.5.9
+- PHP >= 5.5.9 < 7.2.0
 - OpenSSL PHP Extension
 - PDO PHP Extension
 - Mbstring PHP Extension
 - Tokenizer PHP Extension
 </div>
 
+> **Note** recent changes to PHP (version 7.2.0 and above) can, in some situations, cause errors and warnings. If you want to use PHP 7.2.0 and above, consider using a newer version of PHP.
```
Comment (svenluijten) [inline installation.md]: > If you want to use PHP 7.2.0 and above, consider using a newer version of PHP.

Huh?

Comment (assertchris) [reply installation.md]: Oh, silly me!

**Comment by taylorotwell** [comment]
Let's fix the problem instead of saying we don't support 7.2.

**Comment by laurencei** [comment]
@taylorotwell - ping for FYI relevant long discussions here about the 7.2 incompability issues for Laravel 5.1 that were discussed previously; https://github.com/laravel/framework/pull/20258

Not sure how we can approach the fix though.

**Comment by EspadaV8** [comment]
Is the take away here that PRs fixing 5.1 to work with 7.2 would be accepted? If so I'm more than happy to open a few PRs to get things working.

**Pull request closed without merging**